### ISSUE TITLE
Depricate this package

### ISSUE THREAD

**Issue by KronosTheLate** [Otto-AA/dashy-todo#4]
It seems like the  [drafting package](https://typst.app/universe/package/drafting) provides notes in the margin much like this one. Furthermore it seems to fix the overlapping notes issue, have cool sticky-notes, and be more activly maintained. In order not to clutter the "typst universe", I suggest archving this repo, mention drafting in the readme, and removing this package from [typst/universe](https://typst.app/universe).

**Comment by Otto-AA** [comment]
I agree that most users that want a todo functionality should look at the `drafting` package. In fact, I only created this library because I didn't find theirs when I needed it. This is also why I made a PR at the drafting repo to add the `todo` keyword, so now it shows up when searching for "todo".

So, increasing visibility of the `drafting` package seems like a good idea, we can add a reference to `drafting` in the README to make it simpler to find. Does #5 look good to you?

However, I don't see the point of deprecating this package. Imo, it works well enough and it can be good to have alternatives, s.t. when one package breaks in an edge case maybe the other implementation still works. Apart from that, I don't think the typst universe has a way of "deprecating" something, and simply removing it would probably break existing Typst files that use dashy-todo.

**Comment by KronosTheLate** [comment]
Right. I am also not sure what the process is like. My worry was just that having to many options makes it hard to choose the right one, so if it is known that the other package is generally better for most people, I would love it if that fact was made very clear, ideally as early as when searing for "todo" on the typst universe. Currently, one has to open the github repo for each one to see this fact, which is more than should be neccesary. But as I am not sure how to depricate a package, other than perhaps archiving this repo, then the readme-comment in fine for now. Any curious user should find it rather quickly.

**Comment by Otto-AA** [comment]
The new README version should soon be published with this note:

> I suggest to take a look at the [drafting package](https://typst.app/universe/package/drafting), which is a more feature-complete annotation library, but also suitable for simple TODO comments.

I think this is clear enough that you check it out, especially if something is missing from this package.

> My worry was just that having to many options makes it hard to choose the right one, so if it is known that the other package is generally better for most people, I would love it if that fact was made very clear, ideally as early as when searing for "todo" on the typst universe. Currently, one has to open the github repo for each one to see this fact, which is more than should be neccesary.

When searching for "todo", it highlights the drafting package as a featured package. From there, you can view the package descriptions with images and quickly decide which of them you want to use. At least with the additional comment, it should not be hard to choose.

**Comment by KronosTheLate** [comment]
Cool. Thank you!
